microsoft-edge/progressive-web-apps-chromium/whats-new/pwa.md

```diff
@@ -12,6 +12,43 @@ ms.date: 02/23/2022
 This page lists the updates to Progressive Web Apps (PWAs) and Web Apps from the Microsoft Edge Web Apps team.  To try new features, read these announcements.  To stay up to date with the latest and greatest features, download the [Microsoft Edge preview channels](https://www.microsoft.com/edge/download/insider) (Beta, Dev, and Canary).
 
 
+<!-- ====================================================================== -->
+## What's New in Microsoft Edge 114
+
+Starting with Microsoft Edge 114, you can detect when your PWA runs in the sidebar in Microsoft Edge by using User-Agent Client Hints.
+
+To learn more, check out these resources:
+
+* [Detect usage in the sidebar](../how-to/sidebar.md#detect-usage-in-the-sidebar), in _Build PWAs for the sidebar in Microsoft Edge_.
+* [Detecting Microsoft Edge from your website](../../web-platform/user-agent-guidance.md).
+
+
+<!-- ====================================================================== -->
+## What's New in Microsoft Edge 112
+
+Starting with Microsoft Edge 112, Progressive Web Apps can experimentally opt-in to be pinned to the sidebar in Microsoft Edge.
+
+The sidebar in Microsoft Edge allows users to easily access popular websites and utilities alongside their browser tabs. The content in the sidebar augments the user's primary task by enabling side-by-side browsing and minimizing the need to switch contexts between browser tabs.
+
+To make your PWA ready for pinning to the sidebar in Microsoft Edge, use the `edge_side_panel` manifest member.
+
+![Microsoft Edge with one tab opened in a TODO list app, and PWAmp in the sidebar](./pwa-images/using-pwamp-in-sidebar.png)
+
+To learn more, see [Build PWAs for the sidebar in Microsoft Edge](../how-to/sidebar.md).
+
+
+<!-- ====================================================================== -->
+## What's New in Microsoft Edge 108
+
+Microsoft Edge 108 introduces experimental support for Progressive Web Apps to define their own widgets in the Widgets Board in Windows 11.
+
+Widgets are defined in your PWA manifest file, by using the `widgets` manifest member. To make widgets easy to create and adapt to various operating system widgets dashboards in the future, widgets are displayed using Adaptive Cards templates. Microsoft Edge also makes it possible to handle user actions on PWA widgets from your service worker file.
+
+![Windows Widgets Board, next to the PWAmp demo app. The Widgets Board contains the PWAmp mini player widget, showing the current song playing in the PWAmp app](./pwa-images/pwamp-widget.png)
```

Comment:
alt-text pattern: consider moving or copying some of the alt-text into the article body.  The regular view lacks the alt-text, so only a fraction of users benefit from the points that are made in the alt-text.  First try changing alt-text to a lead-in with a colon, and then make the alt-text shorter.
```suggestion
Windows Widgets Board, next to the PWAmp demo app. The Widgets Board contains the PWAmp mini player widget, showing the current song playing in the PWAmp app:

![Windows Widgets Board, next to the PWAmp demo app. The Widgets Board contains the PWAmp mini player widget, showing the current song playing in the PWAmp app](./pwa-images/pwamp-widget.png)
```

rendered section:
https://review.learn.microsoft.com/microsoft-edge/progressive-web-apps-chromium/whats-new/pwa?branch=pr-en-us-2842#whats-new-in-microsoft-edge-108